### DISCUSSION TITLE
Total balance in their currency

### DISCUSSION THREAD

**Discussion by evgewakan** [chr233/ASFEnhance#78] [功能建议 / Ideas]
hello, can you add command for calculate total balance from all accounts in their currency?

example:
Total
USD - 0.00$
EUR - 0.00€
RUB - 0.00₽
etc.

**Comment by SENPAY98K** [comment]
use asf default command `balance asf`

**Reply by evgewakan** [reply]
> use asf default command `balance asf`

i know about this command, but i need view total balance (bot1+bot2+...botN) X.XX$

**Reply by SENPAY98K** [reply]
Idk, it does not seem very usefull but wait for the developer to answer you.
